src/main/java/seedu/address/model/gradebook/GradebookComponent.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.gradebook;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * Represents a component in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+@XmlRootElement(name = "gradebook")
+
+public class GradebookComponent {
+
+    private String gradeItemName;
+    private String moduleCode;
```

Comment:
Annotate the 2 variables here like so: `@XmlElement(name = "<variableName>", required = true, nillable = true)`